web/tests/peer_data.test.js

```diff
@@ -235,6 +242,11 @@ test("get_subscriber_count", () => {
 
     peer_data.remove_subscriber(india.stream_id, george.user_id);
     assert.deepStrictEqual(peer_data.get_subscriber_count(india.stream_id), 1);
+
+    peer_data.add_subscriber(india.stream_id, welcome_bot.user_id);
+    assert.deepStrictEqual(peer_data.get_subscriber_count(india.stream_id), 2);
+    // Get the count without bots
+    assert.deepStrictEqual(peer_data.get_subscriber_count(india.stream_id, false), 1);
```

Comment:
These tests were in the wrong commit, I fixed that in #28848 which should let us merge the changes to several modules not related to the buddy list that this PR requires.